lib/bolt/config.rb

```diff
@@ -9,13 +9,15 @@
 require 'bolt/transport/winrm'
 require 'bolt/transport/orch'
 require 'bolt/transport/local'
+require 'bolt/transport/remote'
 
 module Bolt
   TRANSPORTS = {
     ssh: Bolt::Transport::SSH,
     winrm: Bolt::Transport::WinRM,
     pcp: Bolt::Transport::Orch,
-    local: Bolt::Transport::Local
+    local: Bolt::Transport::Local,
+    remote: Bolt::Transport::Remote,
```

Comment:
This is a larger issue(or two closely related issues) I think we need to discuss and/or research.

Are all remote targets devices(ie physical devices or vms with limited control APIs)?
No, remote tasks should be used for cloud resources, notification APIs etc.

Do they use the puppet-device framework?
Usually but not always. I would expect most ruby implementations to use the puppet-device framework but they could very will be written without that especially to enable non-ruby implementations.

My general hypothesis is that module authors especially ruby authors will want to think of the modules they write as "device" modules since they use the device framework and renaming the framework is probably not worth the effort. On the other hand, I think bolt users, and eventually PE GUI admins should not think of these things as devices but rather remote targets that rely on a proxy since it's weird to call an aws account a device.

Reply:
Naming is hard :) This seems to echo us not knowing how to refer to ephemeral, cloudy | container "nodes". 

Definitely worth more discussion.